types/node/assert.d.ts

```diff
@@ -77,7 +77,28 @@ declare module "assert" {
         function match(value: string, regExp: RegExp, message?: string | Error): void;
         function doesNotMatch(value: string, regExp: RegExp, message?: string | Error): void;
 
-        const strict: typeof assert;
+        const strict: Omit<
```

Comment:
`Omit` doesn’t preserve callability.

Reply:
Good catch - need to add `(value: any, message?: string | Error): asserts value`; I'll do that plus add a test to catch it in the future.